pkg/api/frontendsettings.go

```diff
@@ -539,8 +541,9 @@ func (hs *HTTPServer) getFSDataSources(c *contextmodel.ReqContext, availablePlug
 					JSONData:  ds.JSONData,
 					Signature: ds.Signature,
 					Module:    ds.Module,
-					BaseURL:   ds.BaseURL,
-					Angular:   ds.Angular,
+					// ModuleHash: hs.pluginAssets.ModuleHash(c.Req.Context(), ds),
+					BaseURL: ds.BaseURL,
+					Angular: ds.Angular,
```

Comment:
This is for `ds.BuiltIn`. Do we need the ModuleHash in this case @jackw ? We also don't have the LoadingStrategy in this case, so not sure